int8 state
mines/region region
string objective_type

